sdk/src/types/block/payload/mod.rs

```diff
@@ -80,6 +80,16 @@ impl Payload {
     crate::def_is_as_opt!(Payload: SignedTransaction, TaggedData);
```

Comment:
Just noticing that CandidacyAnnouncement is missing here, can you add it please

Reply:
Added